vomsimporter.py

```diff
@@ -435,6 +439,28 @@ def add_user_to_group(self, iam_user, iam_group):
         r = self._s.patch(url, headers=headers, json=payload)
         r.raise_for_status()
 
+    def remove_user_from_group(self, iam_user, iam_group):
+        logging.debug("Removing user %s from group %s", self.iam_user_str(
+            iam_user), self.iam_group_str(iam_group))
+
+        url = "%s/scim/Groups/%s" % (self._base_url(), iam_group['id'])
+        payload = {
+            'schemas': ['urn:ietf:params:scim:api:messages:2.0:PatchOp'],
+            'operations': [
+                {
+                    'op': 'remove',
+                    'path': 'members',
+                    'value': [{
+                        'display': iam_user['displayName'],
+                        'value': iam_user['id']
+                    }]
+                }]
+        }
+        headers = {'Content-type': 'application/scim+json'}
+
+        r = self._s.patch(url, headers=headers, json=payload)
```

Comment:
What happens if you first remove from group A/B and then from group A/B/C. The former implies the second, so when you issue the second you get an exception, which is not appropriately caught, right?

Reply:
Yes, you are right, I did not took into account IAM automatic cleanup of subgroups